[doc] Clarify doc around the WORKFLOW token creation (#12)
DataDog/sync-upstream-release-tag: README.md

## Usage

> 1. Add a secret called `WORKFLOW_TOKEN` with write access to your repository. This must be a github access token with at least the following permissions (the scope to update the Github Actions is needed in case the upstream repo modify its own actions):

Reviewer: ```suggestion
1. Add a secret called `WORKFLOW_TOKEN` with write access to your repository. This must be a [personal github access token](https://github.com/settings/tokens) with at least the `repo` and `workflow` permissions (the scope to update the Github Actions is needed in case the upstream repo modifies its own actions):
```

Maybe like so? Not sure the perms screenshot is needed

Reviewer: And: Authorize SSO for the Datadog org on the token

Author: unless you are strongly against, let's keep the screen for extra clarification :D 

I'll add a line about the SSO

## Usage

1. Add a secret called `WORKFLOW_TOKEN` with write access to your repository. This must be a [personal github access token](https://github.com/settings/tokens) with at least the `repo` and `workflow` permissions (the scope to update the Github Actions is needed in case the upstream repo modifies its own actions):
   ![token_scope](./token%20scope.png)

> 2. If your org enforce the use of SSO, you must [authorize your token](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)

Reviewer: ```suggestion
2. If your org enforces the use of SSO, you must [authorize your token](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)
```

> 3. Add this to a `.github/workflows/sync.yaml` file in your fork

Reviewer: ```suggestion
3. Add this workflow definition to a `.github/workflows/sync.yaml` file in your fork:
```
